nevergrad/functions/control/mujoco.py

```diff
@@ -23,10 +23,20 @@ class GenericMujocoEnv:
     random_state: int or None
         random state for reproducibility in Gym environment.
     """
-    def __init__(self, env_name, state_mean, state_std, num_rollouts,
-                 random_state):
-        self.mean = np.array(state_mean)
-        self.std = np.array(state_std)
+
+    def __init__(self, env_name, num_rollouts,
+                 random_state, online_stats, state_mean=None, state_std=None):
+        self.online_stats = online_stats
+        self.s_n = 0.
+        self.n_obs = 0
+
+        if self.online_stats:
+            self.mean = 0.
+            self.std = 0.
+        else:
+            self.mean = np.array(state_mean)
+            self.std = np.array(state_std)
```

Comment:
preferably use `np.array(..., copy=True)`. I think this is the default, but I tend to prefer emphasizing it, especially since otherwise you could be modifying a global variable without noticing it